desktop/wasm_exec.js

```diff
@@ -70,7 +70,7 @@
 		const nodeCrypto = require("crypto");
 		global.crypto = {
 			getRandomValues(b) {
-				nodeCrypto.randomFillSync(b);
+				return nodeCrypto.randomFillSync(b);
```

Comment:
I created and issue for this #1136 . 
Because of no return and replacement for the global js. I had an issue with crypto lib for node_modules